src/tools/compiletest/src/common.rs

```diff
@@ -536,6 +537,7 @@ impl TargetCfgs {
         let mut all_abis = HashSet::new();
         let mut all_families = HashSet::new();
         let mut all_pointer_widths = HashSet::new();
+        let mut all_rustc_abis = HashSet::new();
```

Comment:
Can you mention how this is different than `all_abis`?

Reply:
Sure

Reply:
And also added a comment here to mention `TargetCfg::rustc_abi`'s comment.